### PR TITLE
Cherry picking fractional stuff that will not break or change dapp

### PR DESCRIPTION
**Pull request by jordajm**
### Checklist:

- [x] Test your work and double-check to confirm that you didn't break anything
- [x] Ensure all new and existing tests pass
- [x] Format code to be consistent with the project
- (not yet but will in future PR) Wrap any new text/strings for translation

### Description:

This PR will get a lot of the fractional usage stuff into master branch without breaking stuff or changing the UX of the current dapp. The goal here is to reduce the amount of stuff I have to maintain in a feature branch without actually implementing fractional usage yet. 

This code can be more thoroughly reviewed when we're actually implementing fractional usage, but feel free to provide feedback if you'd like to...

This PR pretty much just:
- Adds the calendar.js react component
- Adds the calendar util functions
- Adds calendar CSS
- Adds two NPM dependencies
- Adds some images
- Removes `price` as a required property in housing and services schemas

See https://github.com/OriginProtocol/origin/pull/790 for non-breaking changes to `origin-js`

### CODE REVIEW

**Comment by jordajm** [comment]
Sounds good @wanderingstan thanks - there's not really much to test but if you'd like to pull this branch and confirm that I'm not breaking stuff, give 'er a whirl. See you Monday!

File: origin-dapp/package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "rc-slider": "8.6.1",
     "react": "^16.3.2",
     "react-avatar-editor": "^11.0.2",
+    "react-big-calendar": "^0.19.2",
```
Comment (micahalcorn) [inline origin-dapp/package.json]: For the record, I support the addition of these two dependencies. 👍

**Comment by micahalcorn** [comment]
Seems to work ✅

**Comment by micahalcorn** [comment]
@franckc it's green! 💚

**Pull request merged**